lighthouse-core/audits/manifest-display.js

```diff
@@ -53,9 +53,14 @@ class ManifestDisplay extends Audit {
     return ManifestDisplay.generateAuditResult({
       rawValue: hasRecommendedValue,
       displayValue,
-      debugString: 'Manifest display property should be set.'
+      debugString: this.createDebugString(hasRecommendedValue)
```

Comment:
If there's no value, it might be better not to add the `debugString` property in the first place. It's presence, even if blank, indicates some kind of error.